aisynphys/connectivity.py

```diff
@@ -127,6 +127,10 @@ def measure_connectivity(pair_groups, alpha=0.05, sigma=None, fit_model=None):
         ConnectivityModel subclass to fit Cp profile. If combined with
         sigma the fit will be fixed to that sigma. If None, then fit results
         are ommitted from the results
+    dist_measure : str | 'distance'
```

Comment:
This reads "dist_measure can be either a string or 'distance' ". Better:  `dist_measure : str`

Reply:
oh, I thought that was informing the user what the default value is. I'll change that